core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -897,4 +897,23 @@ class RDDSuite extends FunSuite with SharedSparkContext {
       mutableDependencies += dep
     }
   }
+
+  test("Nested RDDs are not supported (SPARK-5063)") {
```

Comment:
A quick `git grep` suggests that every suite uses its own style and that there's not an obvious dominant style.  I'll just change these tests to the lowercase convention to match RDDSuite, but leave the BroadcastSuite ones as-is.